src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -214,6 +178,7 @@
     "minecraft:music_disc.11": "11 Disc",
     "minecraft:music_disc.13": "13 Disc",
     "minecraft:music_disc.5": "5 Disc",
+    "minecraft:music_disc.relic": "Relic Disc",
```

Comment:
As we should use the disc format, there should not be "Disc" at then end of the titles. If you don't have time for these two changes, I will do the update after your PR